static/app/components/events/interfaces/crashContent/exception/banners/stacktraceBanners.tsx

```diff
@@ -146,25 +73,5 @@ export function StacktraceBanners({stacktrace, event}: StacktraceBannersProps) {
     );
   }
 
-  if (
-    shouldShowCodecovPrompt(organization, stacktraceLink) &&
-    !getPromptStatus(prompt, codecovPromptKey)
-  ) {
-    return (
-      <AddCodecovBanner
-        orgSlug={organization.slug}
-        onClick={() => {
-          trackAnalytics('integrations.stacktrace_codecov_prompt_clicked', {
```

Comment:
cleanup `integrations.stacktrace_codecov_prompt_clicked` if it is unused

Reply:
analytics is in getsentry, will open a diff pr for this